docs/environment.md

```diff
@@ -0,0 +1,53 @@
+# Setting up the common environment
+
+Log in to the LSF cluster, where all data processing must take place. Switch to a common EVA production user instead of your personal account. Then adjust and execute the commands below. They will set up the environment, fetch and build the code.
+
+Notes:
+* The first five variables are installation-specific and are blanked in this repository. You can get the values for the EVA use case from the [private repository](https://github.com/EBIvariation/configuration/blob/master/open-targets-configuration.md).
+* By modifying the `*REMOTE` and `*BRANCH` variables, you can run arbitrary versions of both the main and the VEP pipeline. This is highly useful for development and debugging. By default it fetches master branches of both repositories.
+
+```bash
+# This variable should point to the directory where the clone of this repository is located on the cluster
+export CODE_ROOT=
+
+# Location of Python installation which you configured using build instructions
+export PYTHON_INSTALL_PATH=
+
+# Location of bcftools installation path
+export BCFTOOLS_INSTALL_PATH=
+
+# The directory where subdirectories for each batch will be created
+export BATCH_ROOT_BASE=
+
+# Base path of FTP directory on the cluster
+export FTP_PATH_BASE=
+
+# Base bsub command line for all commands.
+export BSUB_CMDLINE="bsub"
+
+# Setting up Python paths
+export PATH=${PYTHON_INSTALL_PATH}:${PYTHON_INSTALL_PATH}/bin:${BCFTOOLS_INSTALL_PATH}:$PATH
+export PYTHONPATH=${PYTHON_INSTALL_PATH}
+
+# External service paths
+CLINVAR_PATH_BASE="ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar"
+
+export MAIN_REMOTE=origin
+export MAIN_BRANCH=master
+export VEP_REMOTE=origin
+export VEP_BRANCH=master
+
+cd ${CODE_ROOT}
+git fetch ${MAIN_REMOTE}
+git checkout ${MAIN_BRANCH}
+git reset --hard ${MAIN_REMOTE}/${MAIN_BRANCH}
```

Comment:
This will overwrite the _source_ files for the Java parser (if there had been any differences, they would be lost), but not the parser itself, since the actual JAR is not tracked by git.

I have added a clarification to the "Setting up environment" protocol: https://github.com/EBIvariation/eva-cttv-pipeline/pull/99/commits/60873948a2ad963e45dc3b3ec5db549f24724359

Reply:
You're right. I'm not sure why the jar file was missing after I setup the environment.

Reply:
Oh, I'm sorry, that must have been me. About a week ago I had to delete the repository and clone it from scratch, because there were some files in the `.git` subdirectory owned by Gary and Tom (which are previous EVA members who worked on Open Targets), and because of that Git kept giving me permission errors when I tried to switch branches, rebase and such. After doing the repo reset, I forgot to generate the parser classes.